zerver/decorator.py

```diff
@@ -200,7 +200,7 @@ def validate_api_key(request: HttpRequest, role: Optional[str],
         if get_subdomain(request) != Realm.SUBDOMAIN_FOR_ROOT_DOMAIN:
             raise JsonableError(_("Invalid subdomain for push notifications bouncer"))
         request.user = remote_server
-        request._email = "zulip-server:" + role
+        request._issuer_id = "zulip-server:" + role
```

Comment:
I think `_issuer_id` isn't the best possible name for this field.  Maybe `request._requestor_for_logs`?

Reply:
Yeah, I struggled with the name :confounded: Changed now.